ROLES.md

```diff
@@ -325,55 +243,50 @@ be the tech lead in a smaller working group, but they are distinct roles.
 
   - Report. Report current status to the TOC meeting every 6 weeks.
 
-- Holistic responsibility for their working group's [feature
-  tracks](./mechanics/FEATURE-TRACKS.md): tracking, health, and execution.
-
-- Perform issue triage on GitHub.
+- Expected to work to holistically maintain the health of the project through:
 
-- Apply/remove/create/delete GitHub labels and milestones.
+  - Reviewing PRs
 
-- Write access to repo (assign issues/PRs, add/remove labels and milestones,
-  edit project, issues, and PRs, edit wiki, create/delete labels and milestones).
+  - Fixing bugs
 
-- Expected to work to holistically maintain the health of the working group through:
+  - Identifying needed enhancements / areas for improvement / etc.
 
-  - Being a good role model
+  - Execute pay-down of technical debt
 
-  - Be an advocate for the working group inside and outside of the community
+- Design/proposal approval authority over the area / component, though
+  escalation to the technical oversight committee is possible
 
-  - Foster a welcoming and collegial environment
+- Holistic responsibility for their working group's [feature
+  tracks](./mechanics/FEATURE-TRACKS.md): review, tracking, health, and execution.
 
-  - Mentoring and guiding approvers, members, and new contributors.
+- Perform issue triage on GitHub
 
-## Scribe
+- Apply/remove/create/delete GitHub labels and milestones
 
-One of the most underrated roles in open source projects is the role of note
-taker. The importance and value of this role is frequently overlooked and
-underestimated. Since one of the core project values is transparency, we have an
-explicit scribe role to recognize these types of contributions. Working group
-scribes assist the Working Group leads with the mechanical processes around
-Working Group meetings.
+- Write access to repo (assign issues/PRs, add/remove labels and milestones,
+  edit issues and PRs, edit wiki, create/delete labels and milestones)
 
-### Requirements
+- Capable of directly applying `lgtm` and `approve` labels for any PR
```

Comment:
I'd drop this -- I don't think we've needed it in quite a while, and it didn't always work right anyway.

Reply:
```suggestion
```